<ui version="4.0" >
 <class>Form3</class>
 <widget class="QWidget" name="Form3" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>175</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>557</width>
    <height>175</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>557</width>
    <height>175</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Ustawienia</string>
  </property>
  <property name="windowIcon" >
   <iconset>
    <normaloff/>
   </iconset>
  </property>
  <widget class="QCheckBox" name="enabledB" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text" >
    <string>Włącz wtyczkę</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ApplyB" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>146</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>Zastosuj</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="echoB" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>40</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text" >
    <string>Echo</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="voiceB" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>60</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text" >
    <string>Usuwanie głosu</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="skipB" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>120</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text" >
    <string>Przeskakiwanie</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="szybkoB" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>80</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text" >
    <string>Szybciej</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>150</x>
     <y>0</y>
     <width>131</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title" >
    <string>Echo</string>
   </property>
   <widget class="QSlider" name="e1s" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>31</width>
      <height>91</height>
     </rect>
    </property>
    <property name="toolTip" >
     <string>Opóźnienie</string>
    </property>
    <property name="minimum" >
     <number>1</number>
    </property>
    <property name="maximum" >
     <number>1000</number>
    </property>
    <property name="orientation" >
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition" >
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval" >
     <number>100</number>
    </property>
   </widget>
   <widget class="QSlider" name="e2s" >
    <property name="geometry" >
     <rect>
      <x>50</x>
      <y>20</y>
      <width>31</width>
      <height>91</height>
     </rect>
    </property>
    <property name="toolTip" >
     <string>Powtarzanie</string>
    </property>
    <property name="minimum" >
     <number>1</number>
    </property>
    <property name="maximum" >
     <number>100</number>
    </property>
    <property name="orientation" >
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition" >
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval" >
     <number>10</number>
    </property>
   </widget>
   <widget class="QSlider" name="e3s" >
    <property name="geometry" >
     <rect>
      <x>90</x>
      <y>20</y>
      <width>31</width>
      <height>91</height>
     </rect>
    </property>
    <property name="toolTip" >
     <string>Głośność echa</string>
    </property>
    <property name="minimum" >
     <number>1</number>
    </property>
    <property name="maximum" >
     <number>100</number>
    </property>
    <property name="orientation" >
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition" >
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval" >
     <number>10</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="surroundB" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>130</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text" >
     <string>Surround</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>290</x>
     <y>0</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title" >
    <string>Skipping</string>
   </property>
   <widget class="QSlider" name="p1s" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>20</y>
      <width>31</width>
      <height>61</height>
     </rect>
    </property>
    <property name="toolTip" >
     <string>Odtwarzanie</string>
    </property>
    <property name="minimum" >
     <number>2</number>
    </property>
    <property name="orientation" >
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition" >
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval" >
     <number>10</number>
    </property>
   </widget>
   <widget class="QSlider" name="p2s" >
    <property name="geometry" >
     <rect>
      <x>60</x>
      <y>20</y>
      <width>31</width>
      <height>61</height>
     </rect>
    </property>
    <property name="toolTip" >
     <string>Pomijanie</string>
    </property>
    <property name="minimum" >
     <number>2</number>
    </property>
    <property name="orientation" >
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition" >
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval" >
     <number>10</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3" >
   <property name="geometry" >
    <rect>
     <x>290</x>
     <y>90</y>
     <width>111</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title" >
    <string>Usuwanie głosu</string>
   </property>
   <widget class="QRadioButton" name="mniejB" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>25</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>Mniej</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="wiecejB" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>45</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>Więcej</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4" >
   <property name="geometry" >
    <rect>
     <x>410</x>
     <y>0</y>
     <width>141</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title" >
    <string>Odwracanie próbek</string>
   </property>
   <widget class="QRadioButton" name="i0" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>30</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>Nie odwracaj</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="i1" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>60</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>Odwróć lewy</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="i2" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>90</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>Odwróć prawy</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="i3" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>120</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>Odwróć oba</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="wolnoB" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>100</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text" >
    <string>Wolniej</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>enabledB</tabstop>
  <tabstop>echoB</tabstop>
  <tabstop>voiceB</tabstop>
  <tabstop>szybkoB</tabstop>
  <tabstop>wolnoB</tabstop>
  <tabstop>skipB</tabstop>
  <tabstop>e1s</tabstop>
  <tabstop>e2s</tabstop>
  <tabstop>e3s</tabstop>
  <tabstop>surroundB</tabstop>
  <tabstop>p1s</tabstop>
  <tabstop>p2s</tabstop>
  <tabstop>mniejB</tabstop>
  <tabstop>wiecejB</tabstop>
  <tabstop>i0</tabstop>
  <tabstop>i1</tabstop>
  <tabstop>i2</tabstop>
  <tabstop>i3</tabstop>
  <tabstop>ApplyB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
